.github/workflows/surge-preview.yml

```diff
@@ -31,14 +31,14 @@ jobs:
       - name: upload surge service
         id: deploy
         run: |
-          export DEPLOY_DOMAIN=https://debezium-github-io-preview-pr-${{ steps.pr.outputs.id }}.surge.sh
+          export DEPLOY_DOMAIN=https://debezium-debezium-github-io-preview-pr-${{ steps.pr.outputs.id }}.surge.sh
```

Comment:
Are you sure about this? Because the links seem to work in the current form?

Reply:
Deploying it manually as it is not using `afc163/surge-preview@v1` action here.